### DISCUSSION TITLE
About Forge version

### DISCUSSION THREAD

**Discussion by doemty** [UltimatChamp/optimum-fuse#14] [Q&A] [answered]
I am not professional at modding, but follow your modpack.
I am curious about Forge version. I mean why do you not use the latest version?
Is it because of synytra connector or something other?

**Comment by UltimatChamp** [comment, marked as answer]
It's because of **Modrinth App**. Modrinth App does not support versions above `47.2.17`. You can manually use the latest loader if you're using some other launcher.

**Reply by doemty** [reply]
Oh, thank you again <3
